server/src/main/java/com/lunchtime/controllers/EventController.java

```diff
@@ -22,7 +24,43 @@ public EventController(EventService eventService) {
 
     @GetMapping
     public ResponseEntity<?> getAll() {
-        List<Event> result = eventService.findAll(new Date());
+        List<Event> result = eventService.findAll();
+        return new ResponseEntity<>(result, HttpStatus.OK);
+    }
+
+    @GetMapping("/categories/{category}")
+    public ResponseEntity<?> getByCategory(@PathVariable String[] category) {
+        List<Event> result = eventService.findByCategory(category);
+        return new ResponseEntity<>(result, HttpStatus.OK);
+    }
+
+    @GetMapping("date/{date}")
+    public ResponseEntity<?> getByDay(@PathVariable @DateTimeFormat(pattern = "yyyy-MM-dd") Date date) {
+        List<Event> result = eventService.findByDay(date);
+        return new ResponseEntity<>(result, HttpStatus.OK);
+    }
+
+    @GetMapping("dates/from={startDate}&to={endDate}")
+    public ResponseEntity<?> getByDateBetween(
+        @PathVariable("startDate") @DateTimeFormat(pattern = "yyyy-MM-dd") Date startDate,
+        @PathVariable("endDate") @DateTimeFormat(pattern = "yyyy-MM-dd") Date endDate) {
+        List<Event> result = eventService.findByDateBetween(startDate, endDate);
+        return new ResponseEntity<>(result, HttpStatus.OK);
+    }
+
+    @GetMapping("month/{month}")
```

Comment:
I have this event 
```
    {
        "id": 7,
        "restaurantId": 1,
        "date": "2021-08-01T00:00:00.000+0000",
        "image": "https://pixabay.com/photos/audience-concert-music-868074",
        "name": "EVENT NAME 100",
        "category": "tasting",
        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
        "isActive": true
    }
```
but request `localhost:8080/api/events/month/august` returns blank array.
Are you sure that request month without year is needed?

Reply:
Request return only this year events. But really i don't sure that we need sort events by month

Reply:
I understood about current year. I meant, that maybe need to show events from one month for all time
And I think, Require manager can help you in this question :)

Reply:
But I am RM ). I can to show events this year and next, but not all years. I think this will be enough.